modules/appeals_api/lib/appeals_api/notice_of_disagreement_pdf_options.rb

```diff
@@ -30,20 +30,12 @@ def veteran_dob
       dob 'Veteran'
     end
 
-    def claimant_name
-      name 'Claimant'
-    end
-
-    def claimant_dob
-      dob 'Claimant'
-    end
-
     def hearing_type_preference
       @notice_of_disagreement.hearing_type_preference
     end
 
     def contact_info
```

Comment:
Remove this method. We already have `veteran_contact_info` that calls the same method on the NOD.

Reply:
^ after going back and looking, this decision makes sense to me, since contact_info is called at least 10 times, and it really makes the code more readable imo.

Reply:
^ and it turns out we're removing all contact_info references anyway now :)